ext/phar/phar_internal.h

```diff
@@ -233,17 +226,17 @@ enum phar_fp_type {
 /* entry for one file in a phar file */
 typedef struct _phar_entry_info {
 	/* first bytes are exactly as in file */
-	php_uint32               uncompressed_filesize;
-	php_uint32               timestamp;
-	php_uint32               compressed_filesize;
-	php_uint32               crc32;
-	php_uint32               flags;
+	uint32_t               uncompressed_filesize;
+	uint32_t               timestamp;
+	uint32_t               compressed_filesize;
+	uint32_t               crc32;
+	uint32_t               flags;
 	/* remainder */
 	/* when changing compression, save old flags in case fp is NULL */
-	php_uint32               old_flags;
+	uint32_t               old_flags;
```

Comment:
Sorry, what I meant is the alignment in the code. Now the members don't line up nicely, but are off by two spaces ;)